src/content/docs/2/guide/notification.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Notifications
+description: Send native notifications to the user.
+---
+
+import PluginBreadcrumb from '@components/PluginBreadcrumb.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+import Stub from '@components/Stub.astro';
+
+<PluginBreadcrumb
+	version="v2.0.0-alpha.2"
+	github="https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/notification"
+	npm="https://www.npmjs.com/package/@tauri-apps/plugin-notification"
+	cratesio="https://crates.io/crates/tauri-plugin-notification"
+	jsApi="/2/reference/js/notification"
+	docsrs="https://docs.rs/tauri-plugin-notification/2.0.0-alpha.2/tauri_plugin_notification/"
+/>
+
+Send native notifications to your user using the notification plugin.
+
+{/* TODO: Demo component */}
```

Comment:
I wonder if we should just keep these as to-dos for now to really focus on writing the docs?

Reply:
Small argument against that would be that if we don't add the demo component now we'll have to rethink the structure later on, and that can be pretty annoying. So I would add the demo component to all of them, and just in the demo component set the text to something like "We're working on a Tauri app, stay tuned!". Something like that. I dunno. I just don't like having to rethink the structure later on.

Bigger argument for it, we do really need to work on content rather than features, I want to start linking people to the beta site as soon as possible.

Reply:
I also think it makes sense to hold off any even attempting to integrate Tauri until a tad bit closer to beta release. I'm personally not all that concerned with the structure point because we're being fairly diligent with how we're structuring the headings so far. 

Another thought I had is that maybe this demo component should really just be a single button and not a whole large box (such as for this guide it would just be a "send notification" button). I don't think we need to parameterise every option because it really is just meant to be a demo. The user gets to have fun in their own app playing with all the levers maybe? 🤔 

But overall I think it really makes sense to hold off a little later down the road when we have a better idea of how the content starts to take shape

Reply:
We can make it just a button, I cant spontaneously think of anything that would *need* configuring, but it's a pretty nice littoe extra thing to have, like adding your own name to the notification showed or what not. Imagine how most UI frameworks show off components letting you tweak various options in the preview, thats kinda what I'm imagining

But yea we'll hold off on it for now and focus on content 👍🏻